src/controllers/settings.js

```diff
@@ -1,45 +1,49 @@
 /* global app:true Chart:true */
-(function (angular, app) {
+(function (window, angular, app, ipcRenderer, dynamineConfig) {
     'use strict';
     const controller = 'SettingsController';
 
     if (typeof app === 'undefined') throw (controller + ': app is undefined');
 
     //console.log(window);
-    app.controller(controller, ['$scope', '$element', 'ajax', 'toast', 'viewFactory', 'dynamineConfig', function ($scope, $element, ajax, toast, viewFactory, dynamineConfig) {
-        console.log("host");
-        console.log(dynamineConfig.host);
+    app.controller(controller, ['$scope', '$base64', 'ajax', 'toast', 'viewFactory', 'dynamineConfig', 'daemon', function ($scope, $base64, ajax, toast, viewFactory, dynamineConfig, daemon) {
 
         viewFactory.title = 'Settings';
         viewFactory.prevUrl = null;
         $scope.dynamineConfig = dynamineConfig;
+
+
+        //TODO: Cluster if we are still doing it
         $scope.UserLoggingIn = function() {
           toast.success('User Logging in');
         }
+
+        //Daemon Host and Password
         $scope.DaemonToggle = function() {
-            toast.success('Daemon Toggled');
+            $('#DaemonOn').click(function() {
+                if($('#DaemonOn').is(':checked')) { 
+                    daemon.connect();
```

Comment:
Turning on the daemon does not mean establishing a connection. Miners should be started or stopped here.